CMakeLists.txt

```diff
@@ -27,11 +27,15 @@ IF(FLEX_FOUND)
   ENDIF()
 ENDIF()
 
+IF(WIN32)
+  ADD_DEFINITIONS(-DYY_NO_UNISTD_H)
+ENDIF()
+
 FILE(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/c)
 INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR})
 INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR})
 
-ADD_LIBRARY(graphqlparser SHARED
+ADD_LIBRARY(graphqlparser
```

Comment:
Is it really necessary to build as a different flavor of library for Windows? I think this might break the Python/Go examples.